nixio/util/util.py

```diff
@@ -97,7 +97,8 @@ def time_to_str(time):
     :return: string in the form "YYYYMMDDTHHMMSS", where T is the date-time separator
     :rtype: str
     """
-    dt = datetime.utcfromtimestamp(time)
+    # dt = datetime.utcfromtimestamp(time)
```

Comment:
Any reason to keep the old line around as a comment?  I think we can just delete it.

Reply:
a long time ago, it must have made sense ;) removed with the updated version